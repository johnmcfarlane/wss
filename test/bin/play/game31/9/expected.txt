 52  1,11 - YARD
 52  1,11 - YERD
 49  1,11 - YAD
 49  1,11 - YOD
 46  1,11 - DAY
 46  1,11 - DEY
 46  1,11 - DRY
 44  1,11 - DAROO
 43  1,9  - YADE
 43  1,11 - YAR
 43  1,11 - YER
 43  1,9  - YODE
 43  1,11 - YOR
 39  1,11 - DERO
 38  1,9  - DAY
 38  1,9  - DEY
 37  1,11 - ARY
 37  1,11 - RAY
 37  1,9  - RODE
 37  1,11 - ROY
 36  1,11 - ORDO
 36  1,11 - REDO
 35  1,9  - RAY
 35  1,9  - ROY
 34  1,11 - ARED
 34  1,11 - DAO
 34  1,11 - DAR
 34  1,11 - DER
 34  1,11 - DOE
 34  1,11 - DOO
 34  1,11 - DOR
 34  1,11 - OORD
 34  1,11 - ORED
 34  1,11 - ROED
 34  1,11 - ROOD
 33  1,9  - YAD
 33  1,9  - YOD
 32  3,11 - ROOED
 31  1,11 - OAD
 31  1,11 - ORD
 31  1,11 - RAD
 31  1,11 - RED
 31  1,11 - ROD
 30  1,11 - AERO
 30  2,11 - ORDO
 30  2,11 - REDO
 28  2,11 - AY
 28  2,11 - OY
 27  2,11 - ADO
 27  2,11 - EDO
 27 10,1  | IYAR
 27  1,9  - OAD
 27  1,9  - RAD
 27  1,9  - RED
 27  1,9  - ROD
 27  3,9  | YOD
 26  2,9  - AYE
 26  3,11 - YO
 25  1,11 - AER
 25  1,11 - AOR
 25  1,11 - ARE
 25  1,11 - ARO
 25  3,11 - DOO
 25  1,11 - EAR
 25  1,11 - OAR
 25  1,11 - OER
 25  2,11 - ORD
 25  1,11 - ORE
 25  2,11 - RED
 25  2,11 - ROD
 25  1,11 - ROE
 25  1,11 - ROO
 24  3,9  | YOR
 23  3,11 - EDO
 22  2,11 - AD
 22  1,9  - DAO
 22  1,9  - DOE
 22  1,9  - DOO
 22  2,11 - ED
 22  2,11 - OD
 22  2,9  - ODA
 22  2,9  - ODE
 21  2,11 - ARO
 21  2,11 - ROO
 21  3,9  - YA
 21  3,9  - YE
 20  3,11 - DO
 19  1,9  - ROE
 19  1,9  - ROO
 19  3,11 - ROO
 19  6,2  - YADE
 19  6,2  - YODE
 18  2,9  - AY
 18  3,9  | DOA
 18  3,9  | DOE
 18  3,9  | DOO
 18  3,9  | DOR
 18  3,11 - ED
 18  3,11 - OD
 18  8,10 - OLEA
 18  8,10 - OLEO
 18  2,9  - OY
 18  3,9  | ROY
 17  5,2  - ADORE
 17  6,2  - AYRE
 17  3,9  - DA
 17  5,2  - DAROO
 17  3,9  - DE
 17  8,10 - OLD
 17  5,2  - RODEO
 17  6,2  - YARE
 17  6,2  - YORE
 16  2,11 - AE
 16  2,11 - AR
 16  2,6  | DERAH
 16  2,11 - EO
 16  2,11 - ER
 16  2,11 - OE
 16  2,11 - OO
 16  2,11 - OR
 16  2,11 - RE
 16  6,2  | REDYE
 16  1,9  - YA
 16  6,5  - YAUP
 16  1,9  - YE
 16  1,9  - YO
 15  6,2  - DARE
 15  6,2  - DERO
 15  9,10 - LEA
 15  9,10 - LOA
 15  9,10 - LOE
 15  9,10 - LOO
 15  6,2  - ORDO
 15  6,2  - REDO
 15  3,9  | ROD
 15  6,2  - RODE
 14  2,9  - AD
 14 11,8  | ADDY
 14  4,11 - DOER
 14  2,9  - ED
 14 11,8  | EDDY
 14  3,11 - EO
 14  2,9  - OD
 14  3,11 - OO
 14  1,11 - YA
 14  1,11 - YE
 14  1,11 - YO
 13 11,8  | ADRY
 13  6,2  - AERO
 13 10,3  | AY
 13  1,9  - DA
 13  1,9  - DE
 13  2,5  - DEARY
 13  2,5  - DERAY
 13  1,9  - DO
 13  4,11 - DOE
 13  2,5  - DORAY
 13  2,5  - DOREY
 13  3,5  - DORY
 13  3,5  - DRAY
 13  3,5  - DREY
 13  3,9  - OE
 13  8,10 - OLE
 13  6,2  - ORAE
 13  2,5  - READY
 13  7,11 - RESAY
 13  7,5  | YA
 13  3,5  - YARD
 13  3,5  - YERD
 13  7,5  - YUP
 12  6,2  | ADORE
 12  3,5  - ADRY
 12  3,9  | AOR
 12 11,9  | DAY
 12  6,2  | DEARE
 12 11,9  | DEY
 12 10,2  - DI
 12  6,2  | DOREE
 12 11,9  | DRY
 12  6,2  | ERADE
 12  6,2  | ERODE
 12 10,1  | ID
 12  3,10 | OY
 12  6,4  - RAJ
 12  2,5  - RAYED
 12  4,5  | REAVOID
 12  3,9  | ROE
 12  3,9  | ROO
 12  3,5  - YADE
 12  2,5  - YEARD
 12  3,5  - YODE
 11 10,3  | AD
 11  3,5  - AERY
 11  9,8  - COE
 11  9,8  - COO
 11  9,8  - CRE
 11  9,8  - CRO
 11  7,5  | DA
 11  1,11 - DA
 11  1,11 - DE
 11  4,6  | DEVOIR
 11  1,11 - DO
 11  7,5  - DUP
 11  7,2  - DYE
 11  3,5  - DYER
 11  3,5  - OARY
 11  3,5  - OORY
 11  3,5  - ROEY
 11  9,11 - SAY
 11  9,11 - SOY
 11  9,1  | WAY
 11  9,1  | WEY
 11  9,1  | WRY
 11  3,5  - YARE
 11  3,5  - YEAR
 11  9,3  - YOB
 11  3,5  - YORE
 10 11,8  | ADOD
 10  2,5  - ADORE
 10  4,5  - ADY
 10  1,9  - AE
 10  2,9  - AE
 10  4,7  | AVOID
 10  7,2  - AYE
 10  4,5  - DAY
 10  4,5  - DEY
 10  4,5  - DRY
 10  1,9  - EO
 10  2,9  - EO
 10 10,9  | EO
 10  6,11 - EYAS
 10  9,10 - LO
 10  2,5  - OARED
 10  1,9  - OE
 10  2,9  - OE
 10  1,9  - OO
 10  2,9  - OO
 10  2,5  - OREAD
 10  4,7  | OVOID
 10  7,11 - RASED
 10  1,9  - RE
 10  2,5  - ROOED
 10  7,11 - ROSED
 10  7,2  - RYE
 10  9,11 - SYD
  9  7,2  - ADO
  9 10,3  | AE
  9 10,3  | AR
  9  3,5  - ARED
  9  4,5  - ARY
  9  4,7  | AVOIR
  9  3,5  - AYRE
  9  9,3  - DAB
  9 11,9  | DAD
  9  3,5  - DAER
  9  7,2  - DAO
  9  3,5  - DARE
  9  2,5  - DAROO
  9  3,5  - DEAR
  9  9,3  - DEB
  9  3,5  - DERO
  9  4,5  | DERV
  9  9,3  - DOB
  9 11,9  | DOD
  9  7,2  - DOE
  9  3,5  - DOER
  9  7,2  - DOO
  9  3,5  - DOOR
  9  3,5  - DORA
  9  7,2  - EDO
  9  3,5  - EYRA
  9  3,10 | OD
  9  7,2  - ODE
  9  5,11 - OER
  9  3,5  - OORD
  9  3,5  - ORAD
  9  3,5  - ORED
  9  3,5  - OYER
  9  4,5  - RAY
  9  3,5  - READ
  9  3,5  - ROAD
  9  2,5  - RODEO
  9  3,5  - ROED
  9  3,5  - ROOD
  9  4,5  - ROY
  9  4,8  | VOID
  9  9,1  | WAD
  9  9,1  | WED
  9  9,1  | WYE
  9  4,5  - YAD
  9  8,2  - YE
  9  8,2  - YO
  9  4,5  - YOD
  9  2,7  | YODH
  8 11,8  | ADDR
  8  3,5  - ADOR
  8  4,6  | ADV
  8  1,11 - AE
  8  1,11 - AR
  8  7,2  - ARE
  8  7,2  - ARO
  8  5,5  - AY
  8  8,2  - DE
  8  4,6  | DEV
  8  8,2  - DO
  8  4,11 - DO
  8  7,11 - DOSER
  8  4,5  - DYE
  8 11,9  | DYE
  8  6,11 - EARS
  8 11,8  | EDDA
  8 11,8  | EDDO
  8  1,11 - EO
  8  1,11 - ER
  8  6,11 - ERAS
  8  6,11 - EROS
  8  2,7  | EYAH
  8 10,1  | IO
  8  3,5  - ODEA
  8  3,5  - ODOR
  8  1,11 - OE
  8  5,11 - OE
  8 10,2  - OI
  8  1,11 - OO
  8  1,11 - OR
  8  3,5  - ORDO
  8  7,2  - ORE
  8  5,5  - OY
  8  1,11 - RE
  8  3,5  - REDO
  8  3,5  - RODE
  8  7,2  - ROE
  8  7,2  - ROO
  8  9,11 - SAD
  8  9,11 - SED
  8  9,11 - SOD
  8  4,8  | VAIR
  8  2,7  | YEAH
  8  2,8  | YOHO
  7 11,8  | ADOR
  7  8,2  - AE
  7  3,5  - AERO
  7  9,3  - AOB
  7  9,3  - ARB
  7  4,5  - AYE
  7  2,7  | DROH
  7  6,11 - ED
  7  8,2  - EO
  7  4,5  - EYR
  7  4,5  - OAD
  7 11,8  | ODEA
  7 11,8  | ODOR
  7  8,2  - OE
  7  8,2  - OO
  7  3,5  - ORAE
  7  9,3  - ORB
  7  4,5  - ORD
  7  9,3  - RAB
  7  4,5  - RAD
  7  8,2  - RE
  7  9,3  - REB
  7  4,5  - RED
  7  4,6  | REV
  7  9,3  - ROB
  7  4,5  - ROD
  7  4,5  - RYA
  7  4,5  - RYE
  7  9,11 - SYE
  7  9,11 - SYR
  7  4,8  | VAI
  7  4,8  | VEI
  7  9,1  | WAE
  7  9,1  | WAR
  7  9,1  | WOE
  7  9,1  | WOO
  7  9,1  | WRO
  7  6,3  | YADE
  7  2,8  | YAH
  7  4,5  - YAR
  7  4,5  - YEA
  7  6,3  | YEDE
  7  2,8  | YEH
  7  4,5  - YER
  7  3,9  | YO
  7  6,3  | YODE
  7  4,5  - YOR
  7  8,9  | sAE
  7  8,9  | sAO
  7  8,9  | sEA
  6  5,5  - AD
  6  4,5  - ADO
  6 11,8  | ADY
  6  8,11 - ASOR
  6  6,3  | AYRE
  6  2,8  | DAH
  6  4,5  - DAO
  6 11,9  | DAO
  6  4,5  - DAR
  6 11,9  | DAR
  6  4,5  - DEA
  6 11,9  | DEA
  6  4,5  - DER
  6 11,9  | DER
  6  2,9  | DHA
  6  4,5  - DOA
  6 11,9  | DOA
  6  4,5  - DOE
  6 11,9  | DOE
  6  2,8  | DOH
  6  4,5  - DOO
  6 11,9  | DOO
  6  4,5  - DOR
  6 11,9  | DOR
  6  6,4  | DYE
  6  7,11 - DYS
  6  7,11 - EASY
  6  5,5  - ED
  6  2,8  | EDH
  6  4,5  - EDO
  6  6,11 - EO
  6  6,11 - ER
  6  6,3  | EYRE
  6 10,6  | LA
  6 10,6  | LO
  6  9,7  - NA
  6  9,7  - NO
  6  5,5  - OD
  6  4,5  - ODA
  6  4,5  - ODE
  6  3,10 | OE
  6  3,10 | OO
  6  4,11 - OO
  6  3,10 | OR
  6  8,11 - OSAR
  6  2,9  | RHD
  6  7,11 - ROSY
  6  5,5  - YA
  6  6,3  | YARE
  6  5,5  - YE
  6  6,3  | YERE
  6  7,11 - YESO
  6  5,5  - YO
  6  6,3  | YORE
  5 10,3  - AB
  5 11,8  | ADD
  5  4,5  - AER
  5  2,9  | AHO
  5  4,5  - AOR
  5  4,5  - ARE
  5  4,5  - ARO
  5  6,4  | AYE
  5  7,11 - AYS
  5  5,5  - DA
  5  6,3  | DARE
  5  7,11 - DASE
  5  5,5  - DE
  5  6,3  | DERE
  5  5,5  - DO
  5  3,9  | DO
  5  7,11 - DOSA
  5  7,11 - DOSE
  5  6,3  | DREE
  5  4,5  - EAR
  5  4,5  - ERA
  5  6,4  | EYE
  5  4,5  - OAR
  5 11,8  | ODD
  5  7,11 - ODSO
  5  4,5  - OER
  5  2,9  | OHO
  5  2,8  | OOH
  5  4,5  - ORA
  5  4,5  - ORE
  5  2,8  | RAH
  5  4,5  - REA
  5  6,3  | REDE
  5  2,8  | REH
  5  2,9  | RHA
  5  2,9  | RHE
  5  2,9  | RHO
  5  6,3  | RODE
  5  4,5  - ROE
  5  4,5  - ROO
  5  6,4  | RYE
  5  9,11 - SAE
  5  9,11 - SAO
  5  9,11 - SEA
  5  9,11 - SER
  5  9,11 - SOE
  5  9,1  | WE
  5  9,1  | WO
  5  7,11 - YAS
  5  6,4  | YEE
  5  7,11 - YES
  5  7,11 - YRS
  4 11,8  | ADO
  4  7,11 - ADS
  4  5,5  - AE
  4  2,9  | AH
  4  5,5  - AR
  4  7,11 - ARSE
  4  7,11 - DAS
  4  6,4  | DEE
  4  6,4  | DOE
  4  7,11 - DOS
  4  8,11 - DSR
  4 11,8  | EDO
  4  2,9  | EH
  4  5,5  - EO
  4  5,5  - ER
  4  2,10 | HA
  4  2,10 | HE
  4  2,10 | HO
  4  2,10 | HR
  4 11,8  | ODA
  4  6,4  | ODE
  4 11,8  | ODE
  4  7,11 - ODS
  4  5,5  - OE
  4  2,9  | OH
  4  5,5  - OO
  4  7,11 - OOSE
  4  5,5  - OR
  4  6,3  | ORAE
  4  7,11 - RASE
  4  5,5  - RE
  4  7,11 - ROSA
  4  7,11 - ROSE
  4  6,8  | TATE
  4 10,4  - YE
  4  6,5  | YE
  4  5,7  | YI
  3 11,8  | AD
  3  7,11 - AES
  3  6,4  | ARE
  3  7,11 - ARS
  3 11,9  | DA
  3 10,4  - DE
  3  6,5  | DE
  3 11,9  | DE
  3  5,7  | DI
  3 11,9  | DO
  3 11,8  | ED
  3  3,9  | EO
  3  6,4  | ERE
  3  7,11 - ERS
  3  4,10 | ID
  3 11,8  | OD
  3  7,11 - OES
  3  3,9  | OO
  3  7,11 - OOS
  3  6,4  | ORE
  3  7,11 - ORS
  3  8,11 - OSE
  3  7,11 - RAS
  3  6,4  | REE
  3  7,11 - RES
  3  6,4  | ROE
  3  7,11 - ROS
  2 10,4  - AE
  2  6,5  | AE
  2  5,7  | AI
  2  8,11 - AS
  2 10,4  - EE
  2  6,5  | EE
  2  8,11 - ES
  2  4,10 | IO
  2 10,4  - OE
  2  6,5  | OE
  2  5,7  | OI
  2  8,11 - OS
  2 10,4  - RE
  2  6,5  | RE
  2  5,10 | SO
  2  9,11 - SO
