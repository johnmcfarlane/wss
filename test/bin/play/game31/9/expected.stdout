 52  1,11 - YARD
 49  1,11 - YOD
 46  1,11 - DAY
 46  1,11 - DEY
 43  1,11 - YAR
 38  1,9  - DAY
 38  1,9  - DEY
 37  1,11 - RAY
 35  1,9  - RAY
 34  1,11 - DOE
 34  1,11 - DOR
 33  1,9  - YOD
 31  1,11 - RAD
 31  1,11 - RED
 31  1,11 - ROD
 30  2,11 - ORDO
 28  2,11 - AY
 28  2,11 - OY
 27  1,9  - RAD
 27  1,9  - RED
 27  1,9  - ROD
 27  3,9  | YOD
 25  1,11 - EAR
 25  1,11 - OAR
 25  1,11 - ROE
 22  2,11 - AD
 22  2,11 - ED
 22  2,11 - OD
 19  4,2  - ROADEO
 18  2,9  - AY
 18  3,9  | DOE
 18  3,9  | DOR
 18  3,11 - ED
 18  2,9  - OY
 17  5,2  - ADORE
 17  8,10 - OLD
 17  5,2  - RODEO
 17  6,2  - YARE
 17  6,2  - YORE
 16  2,11 - AE
 16  2,11 - AR
 16  2,11 - ER
 16  2,11 - OE
 16  2,11 - OR
 16  6,2  | REDYE
 16  1,9  - YA
 16  6,5  - YAUP
 16  1,9  - YE
 16  1,9  - YO
 15  6,2  - DARE
 15  6,2  - DORE
 15  6,2  - ORDO
 15  6,2  - REDO
 15  3,9  | ROD
 15  6,2  - RODE
 14  2,9  - AD
 14  4,11 - DOER
 14  2,9  - ED
 14 11,8  | EDDY
 14  2,9  - OD
 14  1,11 - YA
 14  1,11 - YE
 14  1,11 - YO
 13  6,2  - AERO
 13 10,3  | AY
 13  1,9  - DE
 13  2,5  - DEARY
 13  2,5  - DERAY
 13  1,9  - DO
 13  4,11 - DOE
 13  3,5  - DORY
 13  3,5  - DRAY
 13  2,5  - READY
 13  7,11 - RESAY
 13  7,5  | YA
 13  3,5  - YARD
 13  7,5  - YUP
 12  6,2  | ADORE
 12 11,9  | DAY
 12 11,9  | DEY
 12 11,9  | DRY
 12  6,2  | ERODE
 12  3,10 | OY
 12  6,4  - RAJ
 12  2,5  - RAYED
 12  3,9  | ROE
 11 10,3  | AD
 11  3,5  - AERY
 11  9,8  - COO
 11  1,11 - DE
 11  4,6  | DEVOIR
 11  1,11 - DO
 11  7,2  - DYE
 11  3,5  - DYER
 11  9,11 - SAY
 11  9,11 - SOY
 11  9,1  | WAY
 11  9,1  | WRY
 11  3,5  - YEAR
 11  9,3  - YOB
 10  1,9  - AE
 10  4,7  | AVOID
 10  7,2  - AYE
 10  4,5  - DAY
 10  4,5  - DEY
 10  4,5  - DRY
 10  6,11 - EYAS
 10  2,5  - OARED
 10  1,9  - OE
 10  2,5  - OREAD
 10  4,7  | OVOID
 10  7,11 - RASED
 10  1,9  - RE
 10  7,11 - RESOD
 10  1,5  - ROADEO
 10  7,11 - ROSED
 10  7,2  - RYE
  9  7,2  - ADO
  9 10,3  | AR
  9  9,3  - DAB
  9 11,9  | DAD
  9  3,5  - DEAR
  9  9,3  - DEB
  9  7,2  - DOE
  9  3,5  - DOER
  9  3,5  - DOOR
  9  3,5  - EYRA
  9  3,10 | OD
  9  7,2  - ODE
  9  3,5  - ORAD
  9  3,5  - OYER
  9  4,5  - RAY
  9  3,5  - READ
  9  3,5  - ROAD
  9  2,5  - RODEO
  9  3,5  - ROOD
  9  4,8  | VOID
  9  9,1  | WAD
  9  9,1  | WED
  9  9,1  | WYE
  9  8,2  - YE
  9  8,2  - YO
  9  4,5  - YOD
  9  2,7  | YODH
  8  1,11 - AE
  8  7,2  - ARE
  8  5,5  - AY
  8  8,2  - DE
  8  4,6  | DEV
  8  8,2  - DO
  8  4,11 - DO
  8  7,11 - DOSER
  8 11,9  | DYE
  8  6,11 - EARS
  8 11,8  | EDDO
  8  6,11 - ERAS
  8  6,11 - EROS
  8  3,5  - ODEA
  8  3,5  - ODOR
  8  1,11 - OE
  8  5,11 - OE
  8  3,5  - ORDO
  8  7,2  - ORE
  8  5,5  - OY
  8  1,11 - RE
  8  3,5  - REDO
  8  7,2  - ROE
  8  9,11 - SAD
  8  9,11 - SOD
  8  4,8  | VAIR
  8  2,7  | YEAH
  7  8,2  - AE
  7  3,5  - AERO
  7  9,3  - ARB
  7  6,11 - ED
  7  2,7  | ODAH
  7 11,8  | ODEA
  7 11,8  | ODOR
  7  8,2  - OE
  7  9,3  - ORB
  7  4,5  - RAD
  7  8,2  - RE
  7  9,3  - REB
  7  4,5  - RED
  7  4,6  | REV
  7  9,3  - ROB
  7  4,5  - ROD
  7  4,5  - RYA
  7  9,1  | WAE
  7  9,1  | WAR
  7  9,1  | WOE
  7  9,1  | WOO
  7  2,8  | YAH
  7  4,5  - YAR
  7  4,5  - YEA
  7  2,8  | YEH
  7  3,9  | YO
  7  8,9  | sAE
  7  8,9  | sEA
  6  5,5  - AD
  6  4,5  - ADO
  6  2,8  | DAH
  6 11,9  | DOE
  6  4,5  - DOR
  6 11,9  | DOR
  6  6,4  | DYE
  6  7,11 - EASY
  6  5,5  - ED
  6  2,8  | EDH
  6  6,11 - ER
  6  6,3  | EYRE
  6 10,6  | LA
  6 10,6  | LO
  6  9,7  - NA
  6  9,7  - NO
  6  5,5  - OD
  6  4,5  - ODA
  6  3,10 | OE
  6  3,10 | OR
  6  8,11 - OSAR
  6  7,11 - ROSY
  6  5,5  - YA
  6  6,3  | YARE
  6  5,5  - YO
  6  6,3  | YORE
  5 10,3  - AB
  5 11,8  | ADD
  5  6,4  | AYE
  5  7,11 - AYS
  5  6,3  | DARE
  5  6,3  | DERE
  5  5,5  - DO
  5  3,9  | DO
  5  6,3  | DORE
  5  7,11 - DOSE
  5  6,3  | DREE
  5  4,5  - EAR
  5  4,5  - ERA
  5  6,4  | EYE
  5  4,5  - OAR
  5 11,8  | ODD
  5  2,9  | OHO
  5  2,8  | OOH
  5  4,5  - ORA
  5  2,8  | RAH
  5  6,3  | REDE
  5  2,9  | RHO
  5  6,3  | RODE
  5  6,4  | RYE
  5  9,11 - SAE
  5  9,11 - SEA
  5  9,11 - SER
  5  9,1  | WE
  5  9,1  | WO
  5  7,11 - YES
  4 11,8  | ADO
  4  7,11 - ADS
  4  2,9  | AH
  4  5,5  - AR
  4  7,11 - ARSE
  4  6,4  | DEE
  4  6,4  | DOE
  4  7,11 - DOS
  4  7,11 - EDS
  4  2,9  | EH
  4  5,5  - ER
  4  2,10 | HA
  4  2,10 | HE
  4  2,10 | HO
  4 11,8  | ODA
  4  6,4  | ODE
  4 11,8  | ODE
  4  7,11 - ODS
  4  2,9  | OH
  4  5,5  - OR
  4  7,11 - RASE
  4  7,11 - ROSE
  4  6,8  | TATE
  4 10,4  - YE
  4  6,5  | YE
  3 11,8  | AD
  3  6,4  | ARE
  3  7,11 - ARS
  3 10,4  - DE
  3  6,5  | DE
  3 11,9  | DE
  3 11,9  | DO
  3 11,8  | ED
  3  6,4  | ERE
  3  7,11 - ERS
  3  4,10 | ID
  3 11,8  | OD
  3  7,11 - OES
  3  6,4  | ORE
  3  7,11 - ORS
  3  8,11 - OSE
  3  7,11 - RAS
  3  6,4  | REE
  3  7,11 - RES
  3  6,4  | ROE
  2 10,4  - AE
  2  6,5  | AE
  2  5,7  | AI
  2  8,11 - AS
  2  8,11 - ES
  2 10,4  - OE
  2  6,5  | OE
  2  5,7  | OI
  2  8,11 - OS
  2 10,4  - RE
  2  6,5  | RE
  2  5,10 | SO
  2  9,11 - SO
